src/app/_components/OrganisationMembersGraphic.tsx

```diff
@@ -0,0 +1,277 @@
+    //amount of circles of radius [memberRadius] that can fit on the circle making up a layer
+    const host_layer1 = Math.floor(
+      Math.PI / Math.asin(memberRadius / (6 * layerRadius))
+    )
+    const host_layer2 = Math.floor(
+      Math.PI / Math.asin(memberRadius / (7 * layerRadius))
+    )
+
+    //divide pi by people, if divided num too small -> add layers
+    if (people > host_layer1 + host_layer2) {
+      pi_division = (2 * Math.PI) / host_layer1
+      pi_division2 = (2 * Math.PI) / host_layer2
+      pi_division3 = (2 * Math.PI) / (people - host_layer1 - host_layer2)
+    } else if (people > host_layer1) {
+      pi_division = (2 * Math.PI) / host_layer1
+      pi_division2 = (2 * Math.PI) / (people - host_layer1)
+    } else {
+      pi_division = (2 * Math.PI) / people
+    }
+
+    //setting up layers
+    for (let i = 0; i < Math.min(people, host_layer1); i++) {
+      host_circles.push({
+        name: i,
+        cx: 85 + 6 * layerRadius * Math.sin(i * pi_division - theta),
+        cy: 85 + 6 * layerRadius * Math.cos(i * pi_division - theta),
+        r: memberRadius
+      })
+    }
+    for (let i = host_layer1; i < host_layer1 + host_layer2; i++) {
+      host_circles.push({
+        name: i,
+        cx: 85 + 7 * layerRadius * Math.sin(i * pi_division2 + theta),
+        cy: 85 + 7 * layerRadius * Math.cos(i * pi_division2 + theta),
+        r: memberRadius
+      })
+    }
+    for (let i = host_layer1 + host_layer2; i < people; i++) {
+      host_circles.push({
+        name: i,
+        cx: 85 + 8 * layerRadius * Math.sin(i * pi_division3 - theta),
+        cy: 85 + 8 * layerRadius * Math.cos(i * pi_division3 - theta),
+        r: memberRadius
+      })
+    }
+    //animating the circles
+    useEffect(() => {
+      const updateTheta = () => {
+        setTheta(prevTheta => (prevTheta + 0.001) % (2 * Math.PI))
+      }
+
+      const interval = setInterval(updateTheta, updateSpeed)
+
+      return () => clearInterval(interval)
+    }, [])
+
+    return host_circles
+  }
+
+  const pgCircles = PgCircleList(20).map(circle => {
+    return (
+      <circle
+        key={circle.name}
+        fill="red"
+        cx={circle.cx}
+        cy={circle.cy}
+        r={circle.r}
+      />
+    )
+  })
+
+  const otCircles = OtCircleList(80).map(circle => {
+    return (
+      <circle
+        key={circle.name}
+        fill="rgb(0 215 144)"
+        cx={circle.cx}
+        cy={circle.cy}
+        r={circle.r}
+      />
+    )
+  })
+
+  const hostCircles = HostCircleList(120).map(circle => {
+    return (
+      <circle
+        key={circle.name}
+        fill="rgb(0 100 70)"
+        cx={circle.cx}
+        cy={circle.cy}
+        r={circle.r}
+      />
+    )
+  })
+
+  return (
+    <figure className="relative mt-5 w-9/12 object-scale-down">
+      <svg viewBox="0 0 170 170" xmlns="http://www.w3.org/2000/svg">
+        <circle fill="orange" cx="85" cy="85" r={memberRadius} />
+        {pgCircles}
```

Comment:
It is good that these are separated by group! One possible improvement for this page is to enable each group separately which will be much easier with this structure.

Reply:
What does enabling a group entail?

E.g
Giving every group of circles an ID and then
svg.getElementById("group_name").disabled = true / false?

Reply:
checked it out, best solution was to set opacity between 1 to 0